components/director/internal/graphql/scalars/helpers.go

```diff
@@ -0,0 +1,52 @@
+package scalars
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+func writeResponse(in interface{}, w io.Writer) error {
+	bytes, err := json.Marshal(in)
+	if err != nil {
+		return errors.Errorf("error with marshalling %T", in)
+	}
+
+	_, err = w.Write(bytes)
+	if err != nil {
+		return errors.Errorf("Error with writing %T", in)
+	}
+	return nil
+}
+
+func convertToString(in interface{}) (string, error) {
+	if in == nil {
+		return "", errors.New("input should not be nil")
+	}
+
+	value, ok := in.(string)
+	if !ok {
+		return "", errors.Errorf("unexpected input type: %T, should be map[string][]string", in)
```

Comment:
This error seems to be a bit misleading here.

Reply:
done